File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task25_CLAUDE_claude-3-5-sonnet-20240620.cpp

#include <vector>
#include <string>

class Solution {
public:
    static std::vector<std::string> number(const std::vector<std::string> &lines) {
        std::vector<std::string> result;
        for (size_t i = 0; i < lines.size(); ++i) {
            result.push_back(std::to_string(i + 1) + ": " + lines[i]);
        }
        return result;
    }
};

File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task25_CLAUDE_claude-3-5-sonnet-20240620.java

import java.util.ArrayList;
import java.util.List;

public class Solution {
    public static List<String> number(List<String> lines) {
        List<String> result = new ArrayList<>();
        for (int i = 0; i < lines.size(); i++) {
            result.add((i + 1) + ": " + lines.get(i));
        }
        return result;
    }
}
